org.eclipse.xtext/src/org/eclipse/xtext/serializer/impl/Serializer.java

```diff
@@ -52,6 +52,7 @@
 
 	@Override
 	public String serialize(EObject obj) {
+		com.google.common.base.Preconditions.checkArgument(obj != null, "obj must not be null.");
```

Comment:
any reason you dont import the type?

Reply:
No, not really, just copy-pasted the method. I just added the import.